### PR TITLE
LRQA-68129

### PR DESCRIPTION
**Pull request by gleicelisbino**

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path [hunk 1]
```diff
@@ -141,9 +141,9 @@
 	<td></td>
 </tr>
 <tr>
-   <td>FIELD_VALIDATION_VALUE</td>
-   <td>//div[contains(@data-field-name,'validation')]//label[contains(text(),'${key_fieldLabel}')]</td>
-   <td></td>
+<td>FIELD_VALIDATION_VALUE</td>
+<td>//div[contains(@data-field-name,'validation')]//label[contains(text(),'${key_fieldLabel}')]</td>
+<td></td>
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path]: @gleicelisbino, it shouldn't be removed.

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path (hunk 1 above)
Comment (gleicelisbino) [reply portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path]: Okay! @natocesarrego

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase [hunk 2]
```diff
@@ -285,8 +285,8 @@ definition {
 			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
 
 		FormFields.editValidation(
-			validationArgument = "Matches",
 			selectOption = "Test Error",
+			validationArgument = "Matches",
```
[the remark this answers is not in the document]
Comment (gleicelisbino) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase:289]: Thaaanks! @natocesarrego

**Comment by natocesarrego** [comment]
ci:forward

**Comment by liferay-forms** [comment]
ci:test:relevant

**Pull request closed without merging**